src/qml/qmlplayerproxy.h

```diff
@@ -140,6 +156,9 @@ class QmlPlayerProxy : public QObject {
     TrackPointer m_pCurrentTrack;
     QmlBeatsModel* m_pBeatsModel;
     QmlCuesModel* m_pHotcuesModel;
```

Comment:
```suggestion
    parented_ptr<QmlCuesModel> m_pHotcuesModel;
```

Reply:
This will require further code change. I will update it in another PR.

Reply:
This line is not added in this PR - and not related to stems.
This is prior QML work, where we decided that we will allow to get less elaborated PRs merged, to accelerate the overall QML development. Thanks Antoine, for your offer to improve this code in a future PR.